book/security.rst

```diff
@@ -1352,6 +1352,15 @@ FriendsOfPHP organization.
     any of your dependencies is affected by a known security vulnerability.
     Therefore, you can easily integrate it in your build process.
 
+.. note::
+
+    To enable the ``security:check`` command, you need to add the
+    `sensio distribution bundle`_ in your composer dependencies.
```

Comment:
While in the context of code I would agree, I see no need in plain documentation where links are all formatted in spaced lower cases format. Unless all other links are changed to follow the CamelCase style.